src/Syntax.jl

```diff
@@ -213,6 +260,27 @@ struct StockAndFlowArguments
     sums::Vector{Symbol}
 end
 
+struct StockAndFlowUArguments
+    stocks::Vector{
+        Pair{
+            Pair{Symbol, Symbol},
+            Tuple{
```

Comment:
Longer term (probably not this PR), it'd be nice to refactor `StockAndFlowF` and `StockAndFlowU` to have less complex argument types, especially re: stocks. I grimace every time I see these.

Reply:
Oh yeah, I agree.  Maybe just make a struct for each of them